src/main/java/com/jetbulb/interviews/NotationConverter.java

```diff
@@ -0,0 +1,7 @@
+package com.jetbulb.interviews;
+
+public class NotationConverter {
```

Comment:
`Smellcode`

Our client in well-defined said that we will be working with many converters of different types.
This class looks like single player and does not show any relation with abstract type, plus abstract type convertor does not exist.